draft-irtf-qirg-principles-03.xml

```diff
@@ -656,6 +676,11 @@
          other schemes the nodes retain greater control over the entangled pair
          generation.</t>
 
```

Comment:
I'm not sure what is actually meant by the paragraph below. What is residual entanglement? Do you mean that whilst photons are transmitted in a directional manner, the resulting entangled pair is not directed?

Reply:
You are correct. I mean that while photons are transmitted in a directional manner, the resulting entangled pair is not directed.